qutebrowser/mainwindow/mainwindow.py

```diff
@@ -492,10 +492,13 @@ def _connect_signals(self):
     @pyqtSlot(bool)
     def _on_fullscreen_requested(self, on):
         if on:
-            self.state_before_fullscreen = self.windowState()
-            self.showFullScreen()
+            self.window_state_before_fullscreen = self.windowState()
```

Comment:
If you really want to rename this, you'll also need to adjust the name in `__init__` and in other places it's used (`browser/commands.py`).